### PR TITLE
[DONE] review translation & clean-up

### PR DESCRIPTION
**Pull request by atefBB**
<div dir="rtl">اصلاح بعض اﻷخطاء النحوية واﻹملائية ومراجعة الترجمة.</div>

### CODE REVIEW

File: content/docs/thinking-in-react.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ redirect_from:
 prev: composition-vs-inheritance.html
 ---
 
-برأينا (React) هو الطريقة الاساسية لبناء تطبيقات ويب كبيرة وسريعة بإستخدام الجافاسكريبت (Javascript) لقد ساعدنا في التوسع في التطبيقات بشكل جيد بفيسبوك (Facebook) وانستجرام (Instagram).
+برأينا React هو الطريقة اﻷساسية لبناء تطبيقات ويب كبيرة وسريعة باستخدام Javascript، لقد ساعدنا في التوسع في التطبيقات بشكل جيد بفيسبوك (Facebook) وانستجرام (Instagram).
```
Comment (mohamedsgap) [inline content/docs/thinking-in-react.md]: you don't have to double write the word twice in Arabic and English like Facebook and Instagram just translate them into arabic only, they are very common in Arabic right now

Comment (atefBB) [reply content/docs/thinking-in-react.md]: I'm not the translator, I'm just a reviewer, I think we should not write them in Arabic!
@3imed-jaberi your thoughts?

Comment (Aissaoui-Ahmed) [inline content/docs/thinking-in-react.md]:
```suggestion
برأينا React هو الطريقة اﻷساسية لبناء تطبيقات ويب كبيرة وسريعة باستخدام JavaScript، لقد ساعدنا في التوسع في التطبيقات بشكل جيد بفيسبوك (Facebook) وانستجرام (Instagram).
```

File: content/docs/thinking-in-react.md [hunk 2]
```diff
@@ -31,27 +31,27 @@ prev: composition-vs-inheritance.html
...
 
-ستري هنا أن لدينا خمس مكونات (components) في تطبيقنا البسيط ولقد قمنا بالكتابة بخط عريض كل مكون ومايمثله من بيانات.
+سترى هنا أن لدينا خمس مكونات (components) في تطبيقنا الصغير، ولقد قمنا بالكتابة بخط عريض في كل مكون وما يمثله من بيانات.
 
   1. **`FilterableProductTable` (باللون البرتقالي):** يحتوي المثال بكامله
   2. **`SearchBar` (باللون الأزرق):** يستقبل ما يدخله المستخدم *(user input)*
-  3. **`ProductTable` (باللون الأخضر):** يعرض و ينقح (filter) مجموعة البيانات *(data collection)* طبقاَ لما ادخله المستخدم *(user input)*
-  4. **`ProductCategoryRow` (باللون الفيروزي):** يعرض عنوان (heading) لكل  *فئة (category)*
+  3. **`ProductTable` (باللون الأخضر):** يعرض وينقح (filter) مجموعة البيانات *(data collection)* طبقاَ لما أدخله المستخدم *(user input)*
+  4. **`ProductCategoryRow` (باللون الفيروزي):** يعرض عنوانا (heading) لكل  *فئة (category)*
```
Comment (Aissaoui-Ahmed) [inline content/docs/thinking-in-react.md]:
```suggestion
  4. **`ProductCategoryRow` (باللون الفيروزي):** يعرض عنوانا (heading) لكل *فئة (category)*
```

**Pull request merged**